distribution/docs/src/main/resources/content/_plugins/nitf-post-process-plugin.adoc

```diff
@@ -0,0 +1,37 @@
+:type: plugin
+:status: published
+:title: NITF Post-Process Plugin
+:link: _nitf_post_process_plugin
+:plugintypes: postprocessplugin
+:summary: Creates and stores the NITF thumbnail and NITF overview images.
+
+The NITF Post-Process Plugin is an experimental replacement for the <<NITF_Render_Plugin,NITF Render Plugin>> that creates and stores the NITF thumbnail and NITF overview images.
+
+===== Related Components to NITF Post-Process Plugin
+
+None.
+
+===== Installing the NITF Post-Process Plugin
+
+The NITF Post-Process Plugin is considered experimental and is not installed by default with a standard installation in the ${alliance-imaging} application.
```

Comment:
✏️ Should probably mention disabling the original plugin if this one is enabled.

Reply:
I think I did include it in the list of steps, but I can add a mention of it in this paragraph as well

Reply:
Oops, sorry I missed it. I think what you have is good.